core/src/test/scala/org/apache/spark/storage/StorageStatusListenerSuite.scala

```diff
@@ -29,9 +29,10 @@ class StorageStatusListenerSuite extends SparkFunSuite {
   private val bm2 = BlockManagerId("fat", "duck", 2)
   private val taskInfo1 = new TaskInfo(0, 0, 0, 0, "big", "dog", TaskLocality.ANY, false)
   private val taskInfo2 = new TaskInfo(0, 0, 0, 0, "fat", "duck", TaskLocality.ANY, false)
+  private val conf = new SparkConf()
 
   test("block manager added/removed") {
```

Comment:
Can you update this test to add checks for the dead executors list you're adding?